app/src/sizes.sass

```diff
@@ -1,5 +1,6 @@
 $contmaxwidth: 960px
 $navbarheight: 40px
+$contpadding: 3ex
```

Comment:
3ex is really not enough. A bare minimum would be 5ex, and you could go up to 8ex.